tilecloud_chain/configuration.py

```diff
@@ -412,23 +412,23 @@ class LayerMapnik(TypedDict, total=False):
 
 # Layer meta
 #
-# Used meta-tiles, default to False, see see https://github.com/camptocamp/tilecloud-chain/blob/master/tilecloud_chain/USAGE.rst#meta-tiles
+# Used meta-tiles, default is False, see https://github.com/camptocamp/tilecloud-chain/blob/master/tilecloud_chain/USAGE.rst#meta-tiles
```

Comment:
Is this configuration to defined whether meta-tiles are to be used or not?
Then it should be
"Use meta-tiles, ..."